Zend/zend_vm_def.h

```diff
@@ -4868,27 +4868,8 @@ ZEND_VM_HANDLER(52, ZEND_BOOL, CONST|TMPVAR|CV, ANY)
 
 ZEND_VM_HANDLER(100, ZEND_GOTO, ANY, CONST)
```

Comment:
We also might just drop it here, make ZEND_GOTO a compile-time only opcode just like ZEND_BRK/ZEND_CONT currently and get a proper fatal about invalid opcode if they slip into VM